### ISSUE TITLE
outputType is ignored when using elements with path that need to be converted to base64

### ISSUE THREAD

**Issue by sijmenvos** [gitbrent/PptxGenJS#414]
If you add images with a path, when saving the presentation it ignores the chosen outputType.

Simple code to reproduce:
```javascript
const PptxGenJS = require('pptxgenjs');

const pptx = new PptxGenJS();
const slide = pptx.addNewSlide();
slide.addImage({
  path: 'https://via.placeholder.com/350x150',
  w: '100%',
  h: '100%',
});

pptx.save('jszip', (result) => {
  console.log(result); // === 'jszip.pptx' file saved locally instead of base64 encoded string
}, 'base64');
```

After converting all images to base64, the outputType isn't passed to the doExportPresentation function:
https://github.com/gitbrent/PptxGenJS/blob/9f23695edd96752052d3fff1ba70ed373d8b5b57/dist/pptxgen.js#L2136

**Comment by sijmenvos** [comment]
Found a workaround if you want to export type to a nodebuffer; if the export name starts with 'http', it will return as 'nodebuffer' type:

```javascript
const PptxGenJS = require('pptxgenjs');

const pptx = new PptxGenJS();
const slide = pptx.addNewSlide();
slide.addImage({
  path: 'https://via.placeholder.com/350x150',
  w: '100%',
  h: '100%',
});

pptx.save('http', (result) => {
  console.log(result); // === Buffer<>
}, 'nodebuffer');
```

**Comment by gitbrent** [comment]
Hi @sijmenvos 

This is fixed in version `3.0`
